sw/device/silicon_creator/lib/manifest.h

```diff
@@ -211,21 +218,22 @@ typedef struct manifest {
   uint32_t entry_point;
 } manifest_t;
 
-OT_ASSERT_MEMBER_OFFSET(manifest_t, signature, 0);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, usage_constraints, 384);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, modulus, 432);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, address_translation, 816);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, identifier, 820);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, length, 824);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, version_major, 828);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, version_minor, 832);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, security_version, 836);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, timestamp, 840);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, binding_value, 848);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, max_key_version, 880);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, code_start, 884);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, code_end, 888);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, entry_point, 892);
+OT_ASSERT_MEMBER_OFFSET(manifest_t, spx_signature, 0);
+OT_ASSERT_MEMBER_OFFSET(manifest_t, rsa_signature, 7856);
```

Comment:
It would be nice but I added it at the start on purpose: this way don't need to touch the RSA signature verification code. We will revisit this when we work on the manifest again.